app/views/sources/edit.html.haml

```diff
@@ -0,0 +1,13 @@
+%h1 Edit Source
+
+= form_for source do |f|
+  .form-group
+    = f.label :name
+    = f.text_field :name, { class: 'form-control', required: true }
+
+  - for type in [:email_rank, :location_rank, :organization_name_rank, :phone_number_rank, :website_rank]
+    .form-group
+      = f.label type
+      = f.select(type, source_rank_options(type), {}, { class: 'form-control' })
```

Comment:
Q: I'm assuming `source_rank_options` is a helper but I didn't see it in ActionView::Helpers::FormOptionsHelper. Do you know where does it come from?